storage/internal/Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: all
+
+all:
+	go fmt ./...
+	cp redis6/driver.go redis7/driver.go
```

Comment:
Lets add a README.md in this location and document the development and testing steps needed to modify or add a new storage driver.